evm/src/main/java/org/hyperledger/besu/evm/worldstate/AbstractWorldUpdater.java

```diff
@@ -97,6 +98,10 @@ public EvmAccount getAccount(final Address address) {
 
   @Override
   public void deleteAccount(final Address address) {
+    Account removed = get(address);
+    if (removed != null && removed.hasCode()) {
+      JumpDestCache.getInstance().invalidate(removed.getCodeHash());
```

Comment:
This purge is unneeded.  If accounts A,B,C,... share the same code Create A, Create B, Delete B, Create C, Delete C, etc.  Even though B and later are deleted A remains and would benefit from the cached jumpdests.  I think we should rely on the the cache's expiration policy instead of pre-deleting (unless we have reference counting handy).